sales_team_security/tests/test_sales_team_security.py

```diff
@@ -34,6 +51,17 @@ def setUpClass(cls):
                 (4, cls.env.ref("sales_team.group_sale_salesman").id)],
             "sale_team_id": cls.team.id,
         })
+        cls.partner_contact_sale_info_propagation_installed = cls._is_module_installed(
```

Comment:
You should make tests independently from this module. If it's interfering, then the problem in in the other module, and would require a mitigation method like this one:

https://github.com/OCA/bank-payment/blob/3c96fedbb95060c9cd3a2aa87d383324d486bee1/account_payment_transfer_reconcile_batch/models/payment_order.py#L28

Reply:
I understand what you say and your proposal, but I don't really know how to interfere with it.
The functionality that is added in this PR auto-add followers of the salesmans (`user_id`) of the contacts. (only in company contacts). The tests check this and work fine.
The `partner_contact_sale_info_propagation` addon auto-define salesman (`user_id`) for all parent contacts, which causes the expected result of the tests to change. Any proposal?

Reply:
Yes, to avoid such propagation on test mode if you are not testing the other module, which style is the link I sent you.